rules/evasion/packer/zstd.yara

```diff
@@ -7,9 +7,9 @@ rule custom_zstd : suspicious {
     strings:
 		// Thanks to https://github.com/VaccinatorSec/yara-rules/blob/master/rules/compressed.yar
 		$ref = {28 B5 2F FD}
-		$ref2 = "zstd"
+		$ref2 = "ZSTD_decompressStream"
     condition:
 		// matches elf or macho
-	    filesize < 52428800 and (uint32(0) == 1179403647 or uint32(0) == 4277009102 or uint32(0) == 3472551422 or uint32(0) == 4277009103 or uint32(0) == 3489328638 or uint32(0) == 3405691582 or uint32(0) == 3199925962) and $ref and $ref2 and math.entropy(1200,filesize) > 6
+	    filesize < 64MB and (uint32(0) == 1179403647 or uint32(0) == 4277009102 or uint32(0) == 3472551422 or uint32(0) == 4277009103 or uint32(0) == 3489328638 or uint32(0) == 3405691582 or uint32(0) == 3199925962) and $ref and $ref2 and math.entropy(1200,filesize) > 7.5
```

Comment:
And the reason why this can't be:
```
smallBinary and $ref and $ref2 and math.entropy(1200,filesize) > 7.5
```

Is because it breaks yara command line?

Reply:
yeah; cross-file dependencies are generally frowned upon in the YARA world because it breaks rule portability. I haven't tested it, but it's possible it would break bincapz as well. :/